src/main/java/org/osc/manager/ism/entities/PolicyEntity.java

```diff
@@ -42,6 +46,9 @@
     @JoinColumn(name = "domain_fk", nullable = false, foreignKey = @ForeignKey(name = "FK_PO_DOMAIN"))
     private DomainEntity domain;
 
+	@ManyToMany(fetch = FetchType.LAZY, mappedBy = "policies")
```

Comment:
Can you make it EAGER? otherwise we will have exceptions while accessing the DB from REST layer.